README.md

```diff
@@ -238,15 +238,41 @@ Similarly, if your application requires on-download progress logic, pytube expos
 
 pytube also ships with a tiny cli interface for downloading and probing videos.
 
-Let's start with downloading:
+Let's start with downloading (it will download the mp4 stream with the highest resolution by default):
 
 ```bash
-$ pytube http://youtube.com/watch?v=2lAe1cqCOXo --itag=22
+$ pytube https://www.youtube.com/watch?v=aqz-KE-bpKQ
 ```
+
 To view available streams:
 
 ```bash
-$ pytube http://youtube.com/watch?v=2lAe1cqCOXo --list
+$ pytube https://www.youtube.com/watch?v=aqz-KE-bpKQ --list
+```
+
+To download a specific stream, use the itag
+
+```bash
+$ pytube https://www.youtube.com/watch?v=aqz-KE-bpKQ --itag=22
+```
+
+To get a list of all subtitles (caption codes) 
+
+```bash
+$ pytube -c -v https://www.youtube.com/watch?v=hsQi4ouYYzI
 ```
 
+To get a download a specific subtitle (caption code) - in this case the english subtitles (in srt format)
```

Comment:
Looks like you made an incomplete wording change here, maybe you meant `To download a specific subtitle [...]`?